setup.py

```diff
@@ -5,12 +5,7 @@
       version='1.0a1',
       install_requires=[
             'sbol3',
-            'rdflib>=5.0.0',
-            'rdflib-jsonld>=0.5.0',
-            'sparqlwrapper>=1.8.5',
-            'pyshacl>=0.13.3',
-            'python-dateutil>=2.8.1',
-            'sbol-factory',
+            'sbol-factory==1.0a6',
```

Comment:
Are all of these other dependencies deleted because they automatically come in via sbol-factory?

Reply:
yes, because the SBOLFactory setup.py was originally forked straight out of this repo, resulting in some redundant specification and danger of the requirements diverging